src/main/java/com/exadel/sandbox/dto/FloorDto.java

```diff
@@ -0,0 +1,24 @@
+package com.exadel.sandbox.dto;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
```

Comment:
FieldDefaults for not using the private key. If you use private you don't need to put fieldDefaults.